plugins/modules/packaging/os/snap_alias.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2021, Alexei Znamensky (russoz) <russoz@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: snap_alias
+short_description: Manages snap aliases
+version_added: 4.0.0
+description:
+    - "Manages snaps aliases."
+options:
+    state:
+        description:
+            - Desired state of the alias.
+        type: str
+        choices: [ absent, present ]
+        default: present
+    name:
+        description:
+            - Name of the snap.
+        type: str
+    alias:
+        description:
+            - Aliases to be created or removed.
+        type: list
+        elements: str
+        aliases: [aliases]
+
+author:
+    - Alexei Znamensky (@russoz) <russoz@gmail.com>
+
+seealso:
+    - module: community.general.snap
+'''
+
+EXAMPLES = '''
+# Install "foo" and "bar" snap
+- name: Create snap alias
+  community.general.snap_alias:
+    name: hello-world
+    alias: hw
+
+- name: Create multiple aliases
+  community.general.snap_alias:
+    name: hello-world
+    aliases:
+      - hw
+      - hw2
+      - hw3
+    state: present   # optional
+
+- name: Remove one specific aliases
+  community.general.snap_alias:
+    name: hw
+    state: absent
+
+- name: Remove all aliases for snap
+  community.general.snap_alias:
+    name: hello-world
+    state: absent
+'''
+
+RETURN = '''
+snap_aliases:
+    description: The snap aliases after execution
+    type: list
+    elements: str
+    returned: always
```

Comment:
Also in check mode? (Because you're not checking that in the check mode tests.)

Reply:
In check mode the list reflects the "before" state. Just updated the description with that info.